docs/zig-package-manager.org

```diff
@@ -0,0 +1,54 @@
+** Zig package manager
+Since 0.11, Zig has now an official built-in package manager named.
+
+There is no global repository like crates.io(Rust) or npmjs.com(Node.js) for Zig packages. Instead, Zig packages are distributed as git repositories.
```

Comment:
Maybe for creates.io and npmjs provides a link to their websites

Reply:
done